packages/blitz-next/src/index-server.ts

```diff
@@ -133,59 +139,13 @@ export const setupBlitzServer = ({plugins}: SetupBlitzOptions) => {
 
 export interface BlitzConfig extends NextConfig {
   blitz?: {
-    resolverBasePath?: ResolverBasePath
+    resolveResolverPath?: ResolverPathOptions
```

Comment:
I think I prefer `resolverPath`. `resolveResolverPath` strongly suggests that it can be a function while in most cases (I think) it will be a string value.

Reply:
I agree that in most cases it will likely be a string, but I don't necessarily agree that it suggests it should be a function. In my eyes it's just a more clear way to say how you want each file path to be resolved. 

But I don't really mind what which one it should be, won't really make a difference to me. 

I am currently on vacation and not in the position to make any changes to this PR, so if you want to make this chance so it can at least be merged and added to a new alpha, then feel free.